### ISSUE TITLE
Issue when working with Azure and OpenAI Callback

### ISSUE THREAD

**Issue by Yanni8** [langchain-ai/langchain#14912]
### System Info

Langchain: v0.0.350
OS: Linux

### Who can help?

@agola11

### Information

- [ ] The official example notebooks/scripts
- [X] My own modified scripts

### Related Components

- [X] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [X] Callbacks/Tracing
- [ ] Async

### Reproduction

The problem occurs when you use Azure with an GPT 4 Model because the Azure API will always respond with `gpt-4` as the Model name. You can also see this in the official Microsoft documentation. https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/gpt-with-vision#output. It will therefore calculate the wrong price → if you use Turbo will the price will be `x3` as it actually should be.

Code to Reproduce:

```python


llm = AzureChatOpenAI(
     deployment_name="GPT4-TURBO"
)

with get_openai_callback() as cb:
    # Run LLM
    print((cb.total_tokens / 1000) * 0.01, "is instead", cb.total_cost)


```

### Expected behavior

It should return the correct price.

**Comment by jvoyez-clarins** [comment]
@dosu-bot, it seems that the cost of gpt-35-turbo version 1106 is missing from the openai_info.py file.
```  
# Azure GPT-35 input
"gpt-35-turbo": 0.0015,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0301": 0.0015,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0613": 0.0015,
"gpt-35-turbo-instruct": 0.0015,
"gpt-35-turbo-16k": 0.003,
"gpt-35-turbo-16k-0613": 0.003,
# Azure GPT-35 output
"gpt-35-turbo-completion": 0.002,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0301-completion": 0.002,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0613-completion": 0.002,
"gpt-35-turbo-instruct-completion": 0.002,
"gpt-35-turbo-16k-completion": 0.004,
"gpt-35-turbo-16k-0613-completion": 0.004,
```

**Comment by JulietteV** [comment]
@dosu-bot, it seems that the cost of Azure gpt-35-turbo version 1106 is missing from the openai_info.py file.
```  
# Azure GPT-35 input
"gpt-35-turbo": 0.0015,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0301": 0.0015,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0613": 0.0015,
"gpt-35-turbo-instruct": 0.0015,
"gpt-35-turbo-16k": 0.003,
"gpt-35-turbo-16k-0613": 0.003,
# Azure GPT-35 output
"gpt-35-turbo-completion": 0.002,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0301-completion": 0.002,  # Azure OpenAI version of ChatGPT
"gpt-35-turbo-0613-completion": 0.002,
"gpt-35-turbo-instruct-completion": 0.002,
"gpt-35-turbo-16k-completion": 0.004,
"gpt-35-turbo-16k-0613-completion": 0.004,
```

**Comment by Yanni8** [comment]
@hwchase17 any update on this issue?

**Comment by PolimiGiovanniArriciati** [comment]
It was fixed in the commit: ed58eeb9c50b16c9927b6c395518cf610c963f5d

as you can see in the master:
https://github.com/langchain-ai/langchain/blame/b20c2640dac79551685b8aba095ebc6125df928c/libs/community/langchain_community/callbacks/openai_info.py#L39

**Comment by spacepirate0001** [comment]
Having the same issue with Meta-Llama-3-70B-Instruct! 
`{"cost":0.0,"total_tokens":0}`
